R/plot.R

```diff
@@ -153,6 +174,12 @@ plot.estimate_R <- function(x, what = c("all", "incid", "R", "SI"),
     options_R$col <- options_R$col[1]
   } else {
     multiple_input <- TRUE
+    if(length(unique(sapply(x, function(e) nrow(e$R)))) > 1) 
```

Comment:
sapply is discouraged in packages because its output is not guaranteed to be the same. Use this instead:

```r
if (length(unique(vapply(x, function(e) nrow(e$R), integer(1)))) > 1)
```

Reply:
thanks for the reminder! Changed as you suggested